src/LivewireManager.php

```diff
@@ -165,7 +165,8 @@ protected function javaScriptAssets($options)
     {
         $jsonEncodedOptions = $options ? json_encode($options) : '';
 
-        $appUrl = config('livewire.asset_url', rtrim($options['asset_url'] ?? '', '/'));
+        $appUrl = rtrim($options['app_url'] ?? config('livewire.app_url'), '/');
```

Comment:
On it's own, that didn't work for me, I changed that line to this:

```php
        $appUrl = $options['app_url'] ?? config('livewire.app_url');
        $appUrl = rtrim($appUrl ?? config('app.url', '/'), '/');
```

Reply:
@ryangjchandler could you also set `'app_url' => config('app.url')` in livewire config?

Reply:
That may work(?) but I don't believe that is a good practice to suggest.

If it does work, it only works because config files *happen* to be processed alphabetically. Also, I don't know how well that works when config caching is taking into consideration.

I feel like sensible fallbacks are the path forward for a good out of the box experience... or let the default value be `env('LIVEWIRE_APP_URL', null)`

Reply:
*sidenote: if you google `Laravel config within config` you get a general sense of why that's not a generally acceptable practice, even though it is not explicitly ever discussed.

Reply:
I see - yeah, relying on load order is probably not a good idea. In the end I'd like to hear @calebporzio's thoughts on the PR before moving forward.